pkg/naming/constants.go

```diff
@@ -297,3 +304,14 @@ const (
 	ScyllaDBManagerTaskRepairSmallTableThresholdOverrideAnnotation = "internal.scylla-operator.scylladb.com/scylladb-manager-task-repair-small-table-threshold-override"
 	ScyllaDBManagerTaskStatusAnnotation                            = "internal.scylla-operator.scylladb.com/scylladb-manager-task-status"
 )
+
+const (
+	// ScyllaDBDatacenterNodesStatusReportSelectorLabel is used to uniformly label nodes status reports created for a ScyllaDB cluster.
+	// It allows easy selection of all datacenter reports for a given cluster.
+	ScyllaDBDatacenterNodesStatusReportSelectorLabel = "scylla-operator.scylladb.com/scylladb-datacenter-nodes-status-report-selector"
+
+	// ForceProceedToBootstrapAnnotation allows to force the bootstrap barrier to proceed without waiting for normal preconditions to be satisfied.
+	// If set to "true", bootstrap barrier will proceed to bootstrap immediately.
+	// This can be used on a particular member Service to override the normal bootstrap precondition checks, or on the entire ScyllaDBDatacenter to override the default behavior for all members.
+	ForceProceedToBootstrapAnnotation = "scylla-operator.scylladb.com/force-proceed-to-bootstrap"
```

Comment:
Does it have any other use than skipping the bootstrap sync in E2Es as a workaround for the limitation we have that prevents us from tweaking the feature gate from a test suite?

Couldn't we deduct the Scylla version at the level of the bootstrap barrier and decide based on that whether the feature is supported?